dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloaderUtils.java

```diff
@@ -347,7 +347,19 @@ protected static String formatAndCheckRepositoryPassword(String pwd) throws Inst
                 throw new InstallException(Messages.PASSWORD_UTIL_MESSAGES.getLogMessage("PASSWORDUTIL_CYPHER_EXCEPTION"));
             }
         }
+    }
 
+    /**
+     * @param pwd
+     * @return
+     */
+    protected static String formatRepositoryPassword(String pwd) {
+        if (!pwd.startsWith("{")) {
```

Comment:
what if the password actually starts with `{`?

We should just exhaustively check for each valid string at the start such as `{xor}`. 
It's still not perfect becuase I guess our users could have a password that starts with `{xor}` too but it's less likely i think.

Reply:
Also, i think if users hav ea password that happens to start with {xor} then liberty was never working for them to begin with